src/utils/aotf.js

```diff
@@ -183,7 +183,8 @@ export const primaryMutations = {
     'release',
     'trigger',
```

Comment:
Trigger is already on the list.

Edit runtime is a bit more specialist so probably reasonable to put behind a button. The idea behind the button is to avoid clutter and keep the interface simple for less advanced users.

Reply:
> Trigger is already on the list.

Duh, I'm blind